src/components/pages/donation_form/subpages/AddressPageDonationReceipt.vue

```diff
@@ -35,6 +35,8 @@
 			</template>
 		</EmailField>
 
+		<MailingListField v-model="mailingList"/>
```

Comment:
a "mailing list" for me means that a group of people can share information within their group via e-mail (1 central address for the mailing list).
what we have is a newsletter, so I'd rather stick with that naming.

Reply:
What we have is exactly this: https://en.wikipedia.org/wiki/Mailing_list. AFAIK the primary use for it is to email people asking them to donate during the campaign, so it's not even really a newsletter.

Reply:
I just realised, I think you're thinking of a mailing group?